code/SCINGE.m

```diff
@@ -12,9 +12,14 @@
 % Outputs:
 % ranked_edges = ranked list of gene interactions with corresponding SCINGE scores
 % gene_influence = ranked lists of regulators (genes) with corresponding SCINGE influence
+SCINGE_version = '0.1.0';
+display(SCINGE_version);
 for rep = 1:num_replicates
     for ii = 1:length(param_list)
-        GLG_Instance(Data,'lambda',param_list{ii}.lambda,'dT',param_list{ii}.dT,'num_lags',param_list{ii}.num_lags,'kernel_width',param_list{ii}.kernel_width,'prob_zero_removal',param_list{ii}.prob_zero_removal,'replicate',rep,'ID',param_list{ii}.ID,'outdir',outdir);
+        if ~isfield(param_list{ii},'prob_remove_samples')
```

Comment:
Why does this need to be handled differently than the other parameters?  Should we also set it in `SCINGE_Example`?

Reply:
We may also want to add it to the hyperparameter list in the main readme.

Reply:
There were two options - one was to introduce the user to the parameter and ask them to enter value as 0.2 by default. The other was to introduce the user to the parameter (which we should add anyway) and claim that the default value is 0.2. Thus, the user doesn't need to explicitly enter the parameter if they want to simply use the default setting.

For the other parameters, we are explicitly asking the users to enter values related to the GLG parameters. I hope to eventually evolve the code to provide informative error messages to the user if they haven't submitted complete params structures.

Reply:
My thinking is that `prob_zero_removal` is similar to `prob_remove_samples` and they should be handled the same way.  If the initial version of SCINGE requires a user to enter a value for `prob_zero_removal`, we can ask the same for `prob_remove_samples`.

This would all be free to change in future versions.

Reply:
I will assign value to prob_remove_samples to SCINGE_Example.m to be consistent with the other parameters.